test-apps/imodel-transformer/README.md

```diff
@@ -6,8 +6,40 @@ The application contained within this directory is a test application for transf
 
 Example command line:
 
-`npm start -- --source=<sourceIModelFileName> --target=<targetIModelFileName>`
+`npm start -- --sourceFile=<sourceIModelFileName> --targetFile=<targetIModelFileName>`
 
 To get usage help run:
 
 `npm start -- --help`
+
+## imodel-transformer as a sample application
+
+This application demonstrates the following:
+
+- Using the `IModelTransformer` API
+- Calling the `ContextRegistry` API to get *context* (project or asset) information
+- Querying `IModelHub` to get iModel information
+- How to handle either *briefcase* or *snapshot* iModel in the same application
+- Using `yargs` to handle command line arguments
+
+## imodel-transformer as a support tool
+
+In addition to being an example of how to use the `IModelTransformer` API within an application, `imodel-transformer` is also a useful support tool.
+It can be used to investigate issues reported against the `IModelTransformer` API.
+A common scenario is a generic report (with minimal details) of IModelTransformer throwing an Error before running to successful completion.
+Here are the steps that can be used:
+
+- Ask to be invited to the project/asset context that contains the iModel. This is straightforward in *QA* and *DEV* but may require more legwork for a user iModel in *PROD*.
+- After being invited, your name should show up in the "Team Members" list. If it does not, you may not have the required permissions to pull a briefcase of the iModel.
+- Get the GUID of the contextId and the GUID of the iModelId. Both should be available in the URL used by Design Review to view the iModel.
+- Optionally, you can turn on additional logging in `Main.ts` and rebuild

```

Comment:
we should probably but the additional logging behind runtime a verbosity flag and not require a rebuild

Reply:
(this may be out of scope, but is worth tracking imo)

Reply:
I like the "verboseLogging" suggestion. I will add that.